### ISSUE TITLE
SOM 2016-20

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-mapping-pipeline#442]

**Comment by QLLZ** [comment]
Model run:
HASH: `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`

**Comment by QLLZ** [comment]
country data report failed: [log file](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/sbatch_logs/country_data_reports/round2/error/prd_cdr_round2_104229_14.log)

**Comment by QLLZ** [comment]
@javierps Yeah, I changed the grid-level_variance plot and a new issue came up. it's about the w value: the length of w value is different in sf_grid and model.rand. After checking the [log file ](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/sbatch_logs/model_runs/round2/slurm-16599252_37.out) of the model run, i found that only 2 chains successfully finished, chain 3-4 actually faild.

**Comment by eclee25** [comment]
Rerun on dev with `drop_multiyear_adm0: yes` config option

**Comment by QLLZ** [comment]
[Updated config](https://github.com/HopkinsIDD/cholera-configs/blob/master/no_covariate_production_2023/round2/config_prd-2023-rd2_SOM_2016_2020.yml)

**Comment by QLLZ** [comment]
Data pull:
HASH `02faaed1a21116cebc29533c68ec0d99fe78d08f`

**Comment by QLLZ** [comment]
Model run:
HASH `02faaed1a21116cebc29533c68ec0d99fe78d08f`

**Comment by QLLZ** [comment]
[Country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/country_data_reports/SOM_2016-01-01_2020-12-31_prd-2023-rd2_66100579c8ad4df9776e34d677e7dd16.country-data-report.html)

**Comment by eclee25** [comment]
Questioning whether model was run on correct branch with correct taxdat installation as there are still multi-year admin 0 obs in the stan input when these should have been dropped.

Opinion: rerun after pull / reinstallation

**Comment by QLLZ** [comment]
rerun on dev:
HASH; `a4d49235ab2737ed5eaafe23264a831ab9cceef7`

**Comment by QLLZ** [comment]
Model run on dev:
HASH: `a4d49235ab2737ed5eaafe23264a831ab9cceef7 `

**Comment by QLLZ** [comment]
Similar issue as before, only 1 chain succeed. 

[log file](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/sbatch_logs/model_runs/new_round2/error/slurm-17304975_4.out)

**Comment by QLLZ** [comment]
Data pull:
HASH: `23e2dc77a189bac73d6b4359bc83d99dfb99cf2f`

**Comment by QLLZ** [comment]
[3 chains failed due to initial values](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/sbatch_logs/model_runs/new_round2/error/slurm-17481155_0.out)

**Comment by QLLZ** [comment]
[Updated config](https://github.com/HopkinsIDD/cholera-configs/blob/master/no_covariate_production_2023/round3/drop_adm0_censored/config_prd-2023-rd3_SOM_2016_2020.yml)

**Comment by QLLZ** [comment]
Data pull:
HASH: `d6950c7afd2b12c35abe6d5321ddfd31a75b2cb0`

**Comment by QLLZ** [comment]
Model run on dev_drop_adm0_censored
HASH: ` d6950c7afd2b12c35abe6d5321ddfd31a75b2cb0`

**Comment by QLLZ** [comment]
Mode run failed: [log file](https://github.com/HopkinsIDD/cholera-mapping-reports/tree/2023_nocovar_production_runs/sbatch_logs/model_runs/new_round3/error)

**Comment by eclee25** [comment]
After drop_censored_adm0 and drop_multiyear_adm0 were turned on (with drop_censored_adm0_thresh: 1), the model was able to initialize and the outputs converged and look good.

**Comment by eclee25** [comment]
Standard Run but with drop_censored_adm0_thresh: 1

**Comment by QLLZ** [comment]
[Config](https://github.com/HopkinsIDD/cholera-configs/blob/master/July_2023_nocovar_production_runs/exception_runs/config_prd-jul2023_SOM_2016_2020.yml)

Data pull & Model run: 

HASH: `211913e0eb7d570ed62c48bc1ae9ecd6671a6cbc`

**Comment by QLLZ** [comment]
[failed model run log file](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/sbatch_logs/errors/model_run/error1/slurm-18569168_1_SOM_1620.out)

**Comment by QLLZ** [comment]
Rerun on dev_u_combs_fix

HASH: `e56580fa5ddab00293e31ff90139351f80ae3f6c`

**Comment by QLLZ** [comment]
[country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/SOM_2016-01-01_2020-12-31_prd-jul2023_e6baff059213ccc63ded4fe43aaa3c34.country-data-report.html)

**Comment by QLLZ** [comment]
Same high incidence rate map as the other time period. Same pop estimates issue.

Modeled cases look good. Convergency looks good.

**Comment by eclee25** [comment]
All diags look good but pop estimates are way too low. Will adjust in post processing

Approve

**Comment by javierps** [comment]
Sep 2023 Production run: all OK.

Suggestion: accept.

**Comment by eclee25** [comment]
Approve
